### ISSUE TITLE
Availability on F-Droid

### ISSUE THREAD

**Issue by feutl** [OpenbookOrg/openbook-app#118]
Because it is Open-Source, wouldn't it be great to have it available on F-Droid as well?
https://f-droid.org/

Should fit your philosophy of open and not feeding "others" perfectly

**Comment by lifenautjoe** [comment]
Definitely. I'll request a package upload. https://gitlab.com/fdroid/rfp/issues

**Comment by mariovor** [comment]
That would be great because I cannot access the Alpha otherwise. I do not have installed the Google Framework on my LineageOS fueled device.

**Comment by 999eagle** [comment]
Yeah I have the same problem. For now you can either compile the app yourself or just join the Slack linked in the readme, @lifenautjoe uploads APKs there. Although distribution through F-Droid would be nicer ^^

**Comment by mariovor** [comment]
Thanks for the hint, got the apk from Slack (better than nothing...).

**Comment by nickbouwhuis** [comment]
> Definitely. I'll request a package upload. https://gitlab.com/fdroid/rfp/issues

Can you link to the request you submitted? Couldn't find it on F-Droids issues page on Gitlab

**Comment by machiav3lli** [comment]
As the people are searching for a "new" safe Haven/Social Platform, this's more relevant than ever. As noted by @IzzySoft in the issue over at [F-Droid's data](https://gitlab.com/fdroid/rfp/-/issues/961), there's 3 things to be cleared:
[ ] Updating Fastlane data
[ ] Replacing the function call of the versionname/-code with literals as F-Droid still don't support it
[ ] OneSignal -including gsf & hms - could be a big problem, suggestion: a build-flavour using Foreground-based Notification service like [Telegram](https://github.com/Telegram-FOSS-Team/Telegram-FOSS) and [Signal](https://github.com/signalapp/Signal-Android)(At least when no gsf is found)
